host/greatfet/peripherals/led.py

```diff
@@ -28,38 +23,15 @@ def __init__(self, board, led_number):
         # TODO: Validate this?
         self.led_number = led_number
 
-
-    def _led_request(self, request_type):
-        """ Internal function that issues a request to the given board for this LED.
-
-        request_type -- The type of request to be issued. Should be one of the 
-                LED_REQUEST_xxx properties of this class.
-        """
-        self.board.comms._vendor_request_out(vendor_requests.SET_LEDS, index=request_type, value=self.led_number)
-
-
-    def set(self, on=True):
```

Comment:
Have you checked to see that we're not using these function calls anywhere else?